api/v1alpha1/openapi.yaml

```diff
@@ -2426,6 +2470,22 @@ paths:
                 $ref: '#/components/schemas/Error'
 components:
   schemas:
+    DeviceDecommission:
+      type: object
+      properties:
+        decommissionType:
+          type: string
+          enum:
+            - 'DefaultUnenroll'
+          description: 'Specifies the desired decommissioning state of the device'
+        reason:
+          type: string
+          description: 'Reason for decommissioning device'
+        requestedBy:
+          type: string
+          description: 'User or process requesting decommissioning of device'
+      required: 
+        - decommissionType
```

Comment:
Should this be optional with a good default?

Reply:
We decided we do need it. It is no longer required.

Reply:
It should be either required or defaulted. I'd vote for required, so we avoid testing for nil-pointers and so the user must explicitly choose one.